src/test/java/cd/go/contrib/elasticagents/dockerswarm/elasticagent/executors/ServerPingRequestExecutorTest.java

```diff
@@ -32,75 +32,87 @@
 public class ServerPingRequestExecutorTest extends BaseTest {
```

Comment:
The current tests only verify the cleanup of agents from one cluster. Can we also verify the cleanup happens across multiple clusters?